src/main/kotlin/cc/woverflow/chatting/chat/ChatTab.kt

```diff
@@ -26,6 +26,7 @@ data class ChatTab(
     @SerializedName("hovered_color") val hoveredColor: Int?,
     @SerializedName("selected_color") val selectedColor: Int?,
     val prefix: String?,
+    val message: List<String>?,
```

Comment:
can you make this transient to make it not save in the json

Reply:
Sure will do!

Reply:
this still isnt annotated with `@Transient`

Reply:
Is is now, sorry about that.